client/my-sites/current-site/style.scss

```diff
@@ -61,46 +61,33 @@
 .current-site__switch-sites {
 	background: lighten( $gray, 30% );
 	border-bottom: 1px solid lighten( $gray, 25% );
-	font-size: 13px;
 	display: block;
-	padding: 8px 8px 8px 40px;
-	color: darken( $gray, 20% );
+	padding: 8px;
```

Comment:
By applying this to the `<span>` instead of the `<Button>`, there's a small gap around the edge where if the user clicks, the site switcher will not be shown.